### ISSUE TITLE
Decreased complement component 7

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5863]
**Preferred term label:**

Decreased complement component 7
**Synonyms**

Decreased C7

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/9218625/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0004431

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/610102

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Analogously for C6!  https://omim.org/entry/612446

**Comment by pnrobinson** [comment]
C5

**Comment by pnrobinson** [comment]
done, added terms
